### DISCUSSION TITLE
Enhancement: UI fixes

### DISCUSSION THREAD

**Discussion by Flashwalker** [TriliumNext/Trilium#7365] [Q&A] [answered]
### Describe feature

# Full width note interface
Currently, on a Full HD screen, the width of the note content is framed and does not expand to 100% width for some reason.
It would be nice to have 100% width.
![image](https://user-images.githubusercontent.com/2188215/210800325-ba835ce5-1ba8-404b-b3dc-0749a428c06a.png)

# Zooming
At this point the zooming with: <kbd>Ctrl</kbd>+Scroll, <kbd>Ctrl</kbd>+`-`, <kbd>Ctrl</kbd>+`=`
changes the scaling of the whole entire Trilium interface.
Is it possible to bind it for the note font-size only?

**Comment by zadam** [comment, marked as answer]
Hi, this can be changed in Options -> Appearance:

<img width="786" alt="image" src="https://user-images.githubusercontent.com/617641/210809437-cc3f6962-9254-44ef-95ac-add008d0610d.png">
